### ISSUE TITLE
New System Enclosure

### ISSUE THREAD

**Issue by mtfunder** [precision-sustainable-ag/psa-planning#195]
## Rationale ##
- We need to design a new enclosure for a Jetson Nano / NX. The first protype enclosure was very temporary. This enclosure needs to show we are capable of delivering a professional and robust system that will survive in the elements.

## What done looks like ##
- All designed and purchased parts in PartsBox.
- All parts are reviewed and 100% ready to purchase.
- Page in confluence explain the design choices and how to build the assembly.

## Stakeholders / Contacts ##
- @mtfunder 

## Additional Information ##
- Should be made from aluminum.
- Attempting for IP 66 or similar rating.
- Passively cool the Jeston and all other components.
- Allow for Wi-Fi signal to get out.
- Connections for 4x GMSL cameras, Power in, Ethernet out, Aux buttons, CANBUS for Optical Flow sensor.

**Comment by mtfunder** [comment]
3D CAD of Forcer Carrier board available here and added to PartsBox:

https://github.com/forecr/forecr_3d_models/blob/master/DSBOARD-ORNXS/rev-1.11/wttp.STEP

Working on reconfiguring the component layout to work with eh ne carrier board.

**Comment by mtfunder** [comment]
All design work is done, waiting on parts to start test assembling boards and final seal test. Preliminary waterproofness testing sucsessful.

**Comment by mtfunder** [comment]
All design work is done, waiting on parts to start test assembling boards and final seal test. Preliminary waterproofness testing sucsessful.

**Comment by mtfunder** [comment]
Made a new task for assembly so marking this as done.
